lib/prompt_info_functions.zsh

```diff
@@ -10,9 +10,15 @@
 # Dummy implementations that return false to prevent command_not_found
 # errors with themes, that implement these functions
 # Real implementations will be used when the respective plugins are loaded
-function chruby_prompt_info hg_prompt_info pyenv_prompt_info \
-  rbenv_prompt_info svn_prompt_info vi_mode_prompt_info \
-  virtualenv_prompt_info {
+function chruby_prompt_info \
```

Comment:
These are spread out one name per line for more readable diffs.

Adds `jenv_prompt_info`.